README.md

```diff
@@ -1,46 +1,429 @@
+
+#### Default root-level override for all schemes
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  votingMachineParams: {
+    votePerc: 45,
+    ownerVote:false
+  }
+});
 ```
 
-You can also override the default configuration settings with environment variables:
+#### Scheme-specific override
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  schemes: [
+    { name: "SchemeRegistrar" },
+    { name: "UpgradeScheme" },
+    {
+      name: "GlobalConstraintRegistrar",
+      votingMachineParams: {
+        votePerc: 30,
+      }
+    }
+  ]
+});
+```
+
+### Create a new DAO with a non-Universal Controller
+
+Create a DAO using the DAOstack `Controller` contract by passing `false` for `universalController`.
+
 ```javascript
-gasLimit=7000000 node yourprojectindex.js
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  universalController: false
+});
 ```
 
+### Create a new DAO with a non-default DaoCreator scheme
+
+Create a DAO using an alternative `DaoCreator` scheme by passing its address for `daoCreatorScheme`.  The alternative scheme must implement the same interface as the Arc `DaoCreator` contract.
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  daoCreatorScheme: anAddress
+});
+```
+
+### Get a previously-created DAO
+
+Get a previously-created DAO using the avatar address:
+
+```javascript
+const dao = await DAO.at(daoAvatarAddress);
+```
+
+### Get the DAOstack Genesis DAO
+
+The DAOstack DAO is named "Genesis".  Obtain it like this:
+
+```javascript
+const genesisDao = await DAO.getGenesisDao();
+```
+
+## Using the Arc Contracts
+
+Arc.js wraps every Arc contract in a JavaScript "wrapper" class.  Every wrapper class inherits from `ExtendTruffleContract` which provides a common set of functions in addition to all of the functions implemented by the specific Arc contract being wrapped.
+
+### Categories of Arc Contracts
+Arc contracts and associated Arc.js contract wrapper classes can be categorized as follows:
+
+#### Schemes
+* SchemeRegistrar
+* UpgradeScheme
+* GlobalConstraintRegistrar
+* ContributionReward
+* VoteInOrganizationScheme
+* VestingScheme
+
+#### Voting Machines
+* AbsoluteVote
+* GenesisProtocol
+
+#### Global Constraints
+
+* TokenCapGC
+
+#### Others
+* DaoCreator
+
+You can pull those categorizations into your code as follows:
+
+```javascript
+import * as ArcJs from '@daostack/arc.js';
+const arcJsWrappers = await ArcJs.getDeployedContracts();
```

Comment:
this seems like not the best advice. wouldn't you usually pull in the wrapper as you do below and call .deployed()? seems like that should be first

Reply:
If I'm finding the right lines you are referring to, this says nothing about obtaining wrappers, it is only about obtaining the categorizations.  But apparently you misunderstood it, so I'll try to make it more clear.

Reply:
see line 313 for what I did.